public/javascript/pageController.js

```diff
@@ -0,0 +1,134 @@
+(function(module){
```

Comment:
Where's the `.eslintrc`? It's clear that some of you were not actively linting your code.